wandb/fastai/__init__.py

```diff
@@ -92,6 +92,7 @@ def __init__(self,
         # Select items for sample predictions to see evolution along training
         self.validation_data = validation_data
         if input_type and not self.validation_data:
+            random.seed(12345)  # Ensures repeatability of selected samples
```

Comment:
I'm wary to call this in the user code as it will seed the random generator for the rest of the script.  Is there a way to do a "scoped" seed, i.e. `with random.seed(12345)` or manually introspect and store the seed before and reset the global seed to it after?

Reply:
Great point! It seems it's possible to do scoped seed: https://stackoverflow.com/a/12369081/3474490
Let me do a few tests to ensure this will work.

Reply:
This seems to work great!
```
>>> import random
>>> myRandom = random.Random(12345)
>>> random.seed(1)
>>> random.random()
0.13436424411240122
>>> myRandom.random()
0.41661987254534116
```
Let's change the order
```
>>> random.seed(1)
>>> myRandom = random.Random(12345)
>>> myRandom.random()
0.41661987254534116
>>> random.random()
0.13436424411240122
```

Reply:
Do you want to leave the option to change the seed (if we don't like the selected examples)?

Reply:
Sure, let's make it an argument that defaults to "12345" and if it's None we don't seed the generator.

Reply:
Sounds good. Actually the default when creating a Random instance is to pass `None`, which uses the current system time.